\documentclass[10pt]{article}
%\usepackage{anysize}
%\papersize{11in}{8.5in}
%\marginsize{1in}{1in}{.5in}{.5in}
\textwidth = 6.5 in
\textheight = 9 in
\oddsidemargin = 0.0 in
\evensidemargin = 0.0 in
\topmargin = -0.5 in
\headheight = 0.35 in
\headsep = 0.15 in
\topskip = 0 in
\footskip = 0.5 in
\pagenumbering{arabic}
\usepackage{setspace}
\usepackage[usenames]{color}
\usepackage[fleqn]{amsmath}
\usepackage{graphicx}
\usepackage{url}
\usepackage{verbatim}
\usepackage{indentfirst}
\usepackage{booktabs}
\usepackage{multirow}
\usepackage[table]{xcolor}
\usepackage{ragged2e}
\usepackage{xspace}
\usepackage{parskip}
\usepackage{tabulary}
\usepackage[normalem]{ulem}
\usepackage{hyperref}
\hypersetup{pdfborder={0 0 0}, colorlinks=true, urlcolor=blue, linkcolor=black}
\usepackage[format=plain, labelsep=period, justification=raggedright, singlelinecheck=true, skip=2pt, font={footnotesize,sf}, labelfont=bf]{caption}
\usepackage{titlesec}
\usepackage{lastpage}
\usepackage{fancyhdr}
\usepackage{ifthen}
\usepackage{wrapfig}

%\usepackage[round]{natbib}
%\bibliographystyle{evolution}
\usepackage[style=nature]{biblatex}
\bibliography{references}

%% Format headers and footers %%%%%%%%%%%%%%%%%%%%
\pagestyle{fancy}
%\lhead{\ifthenelse{\value{page}=1}{}{\sffamily\footnotesize Jamie Oaks}}
\lhead{\sffamily \emph{\docTitle} \\ Jamie R. Oaks}
%\chead{\ifthenelse{\value{page}=1}{{\scshape \docTitle} \\ Jamie Richard Oaks}{\sffamily\footnotesize \docTitle}}
%\rhead{\ifthenelse{\value{page}=1}{}{\sffamily\footnotesize \today}}
\rhead{\sffamily \today}
\cfoot{\sffamily\footnotesize Page \thepage\ of \pageref{LastPage}}
\renewcommand{\headrulewidth}{0.4pt}
\renewcommand{\footrulewidth}{0pt}

%% Format section titles %%%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand\refname{Peer-reviewed Publications}

\titleformat{\section}[hang]
    {\large\sffamily\bfseries}
    {\S\ \thesection.}{.5em}{}[]
\titlespacing{\section}
    {0mm}{1.0ex plus .1ex minus .1ex}{-0.5ex}

\titleformat{\subsection}[hang]
    {\large\sffamily\itshape}
    {\S\ \thesection.}{.5em}{}[]
\titlespacing{\subsection}
    {0mm}{1.0ex plus .1ex minus .1ex}{-0.5ex}

\titleformat{\subsubsection}[runin]
    {\sffamily\bfseries}
    {\S\ \thesection.}{.5em}{}[.---]
\titlespacing{\subsubsection}
    {\parindent}{0pt}{0pt}
%    {\parindent}{1.0ex plus .1ex minus .1ex}{0pt}

%% Format list environments %%%%%%%%%%%%%%%%%%%%%%%%
\renewcommand{\labelenumii}{\arabic{enumi}.\arabic{enumii}}
\renewcommand{\labelitemi}{$\circ$}

\newenvironment{myEnumerate}{
  \begin{enumerate}
    \setlength{\itemsep}{0.25em}
    \setlength{\parskip}{0pt}
    \setlength{\parsep}{0.5em}}
  {\end{enumerate}}

\newenvironment{myItemize}{
  \begin{itemize}
    \setlength{\leftskip}{-4mm}
    \setlength{\itemsep}{0.25em}
    \setlength{\parskip}{0pt}
    \setlength{\parsep}{0.5em}}
  {\end{itemize}}

%% Basic formatting and spacing %%%%%%%%%%%%%%%%%%%%%
\setlength{\parindent}{0em}
\setlength{\parskip}{0.5em}

%% My functions %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newcommand{\ignore}[1]{}
\newcommand{\addTail}[1]{\textit{#1}.---}
\newcommand{\super}[1]{\ensuremath{^{\textrm{#1}}}}
\newcommand{\sub}[1]{\ensuremath{_{\textrm{#1}}}}
\newcommand{\dC}{\ensuremath{^\circ{\textrm{C}}}}
\newcommand{\tableSubItem}{\addtolength{\leftskip}{1em} \labelitemi \xspace}
\newcommand{\myHangIndent}{\hangindent=5mm}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\newcommand{\docTitle}{Curatorial Statement\xspace}
\begin{document}
\raggedright
\singlespacing

My duty as a collection-based scientist is to \textbf{\textit{advocate and
advance the use, maintainence, and value of natural history collections for
understanding the biodiversity of our planet.}}
The primary challenge of curation is striking a balance between the use and
long-term preservation of research material.
However, I believe we are at the beginning of a major shift in how biodiversity
data is distributed and used by scientist that will meet this challenge and
transform the role of biodiversity collections.

\section*{The Future of Biodiversity Collections}
In the near future, a typical museum specimen will be associated
with a wealth of digital information that will be available in real-time
to scientists around the world.
In addition to the typical field data, there will be a genome sequence
and high-resolution, three-dimensional digital scan and x-ray for each
specimen.
Thus, the entire genetic blueprint and internal and external morphology of 
each specimen will be a few keystrokes away from any computer in the world.
While such a rich digital representation will never supplant the need for
preserving the physical specimen, I believe the data will truly revolutionize
the role of biodiversity collections in science and society.
By greatly reducing the frequency with which specimens need be physically
transported and examined, the longevity of specimens will be extended.
Also, if a specimen is ever damaged or lost, the utility of the specimen will
be largely preserved in its digital form.
Having such rich data available from millions of specimens across hundreds of
institutions will greatly expand the types of research questions that can be
addressed with natural history collections.
Shifting to this role will increase the prominence of collections and clearly
demonstrate the importance of the collections to the public, policy-makers,
funding agencies, and donors.

\section*{The Future is Now}
The future described above may seem distant.
However, during my experience working in natural history collections, I have
realized that there are many significant steps that need to be taken now.
I have served as the Curatorial Assistant (CA) for the Herpetology collections
at the University of Kansas (KU) Biodiversity Institute and Louisiana State
University Museum of Natural Science, the former of which is one of the largest
collections of amphibians and reptiles in the United States.
In the absence of a Collection Manager at both collections, I was responsible
for all aspects of managing the collection, including accessions, cataloging,
database management, loans, import/export permits, and general collection and
wet lab upkeep.
While CA for the KU collection, I made several improvements to the collection
database that have increased the efficiency and accuracy of digitizing
new collection data.

Many natural history specimens are already associated with a wealth of ancillary
data that collection databases are ill-equipped to automatically consolidate,
update, and serve to the community in real-time.
For example, many specimens 
have digital photographs and/or audio recordings,
are associated with publications and sequences on
\href{http://www.ncbi.nlm.nih.gov}{NCBI}, are included in molecular and
morphological datasets deposited in repositories like
\href{http://datadryad.org}{Dryad}, are tips on phylogenetic trees in
\href{http://treebase.org}{TreeBASE}, or better yet, on the
\href{http://opentreeoflife.org}{Open Tree of Life}.
Collection databases need to be modernized so that they can serve
as the central hub of information associated with the Collection's
physical holdings.

There are two key ways to accomplish this: (1) transforming how data is 
collected and added to the database, and (2) adding flexibility and
automation to how ancillary data are incorporated and updated.

\subsubsection*{Transforming data collection}
Too much data is lost or corrupted between collection and database entry.
I am interested in developing portable, flexible, easy-to-use software
that allows real-time digitization of data as it is collected in the field.
The ultimate goal is to have software that can sync information such as
GPS coordinates, photographs, audio-recordings, micro-habitat data, and
weather data from other wireless devices.
We already have electronic devices that can collect much of this data and can
communicate it via technologies such as Bluetooth.
So, much of the infrastructure for this is already in place.

\subsubsection*{Automated updating of ancillary data}
Too often, valuable ancillary data never gets associated with specimens.
It is impractical for collection staff to manually mine databases like NCBI,
GenBank, Dryad, and TreeBASE in an effort to consolidate all of the information
available for every specimen.
I am interested in developing flexible database software that automatically
scans such databases and updates the ancillary data as it comes on line.
The software should do this intelligently, flagging specimens whenever there
are conflicting data (e.g., a phylogenetic placement conflicts with the current
taxonomy).
This will transform the collection database into the most current and
comprehensive source for all data associated with its holdings.
After all, this should be the goal of all collections.

\section*{Museum-Based Education}
Similar to , 
Similar to curation, technology will greatly enhance museum-based education.

% \section*{Collection-Based Research}
% While my primary research questions focus on specific taxa of Southeast Asia
% (see research statement), an equally important aspect of my work includes
% broad collecting of amphibians and reptiles to document the biodiversiy of
% this relatively poorly known region of the world.
% an important aspect of my work will always be broad biodiversity survey stuff.
% Funding from NSF BSFI Nat Geo

% The SNOMNH Herpetology collection is strong in blah
% My research program would expand holdings of Old World blah, specifically
% Southeast Asia.

\end{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
